heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -123,6 +123,44 @@ value specified for `timeout` is greater than `schedule`, intermediate checks
 will not be executed by the scheduler.
 
 [float]
+[[monitor-fields]]
+==== `fields`
+
+Optional fields that you can specify to add additional information to the
+output. For example, you might add fields that you can use for filtering log
+data. Fields can be scalar values, arrays, dictionaries, or any nested
+combination of these. By default, the fields that you specify here will be
+grouped under a `fields` sub-dictionary in the output document. To store the
+custom fields as top-level fields, set the `fields_under_root` option to true.
+If a duplicate field is declared in the general configuration, then its value
+will be overwritten by the value declared here.
+
+[float]
+[[monitor-fields-under-root]]
+==== `fields_under_root`
+
+If this option is set to true, the custom <<monitor-fields>>
```

Comment:
@urso It works because Clint is amazing. I complained about the behavior again yesterday, and he fixed it! https://github.com/elastic/docs/commit/e427db2aa82fb34d0d3dae9a4f24748472dc98a7